qa/suites/fs/basic/omap_limit/10.yaml

```diff
@@ -0,0 +1,5 @@
+overrides:
```

Comment:
Hmm, this increases the number of jobs in the fs suite by almost 100 (
we still run all the jobs in fs, unlike rados).

Also, I'm not sure this will really be exercising anything on many of the workunits, as anything that fits in cache (many/most of them) will not hit the omap fetch path